### DISCUSSION TITLE
Generating Telemetry events to identify Interactive videos uniquely

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-inQuiry/Community#21] [Q&A]
Hello All,

We need to track a [list of metrics](https://project-sunbird.atlassian.net/wiki/spaces/PRD/pages/3108995078/Product+Analytics+Dashboard) for which we are proposing the following changes to the telemetry. Which will help to recognize that the activity was a part of Interactive video .

1. Adding the id of the video to the QuestionSet telemetry
2. Adding a new parameter in all Interactive video telemetry to identify it as an Interactive content
Can you please guide us on how this can be implemented?

@Krishnaj20 @vinukumar-vs @HarishGangula
@coolbung @baljeetsingh

**Comment by coolbung** [comment]
@Xitija -- Which events do you plan to update (eg: START, ASSESS, END etc) ? Request you to propose the updated JSON for these events.
